Makefile

```diff
@@ -160,7 +160,7 @@ fmt: ## Run go fmt against code.
 
 .PHONY: vet
 vet: ## Run go vet against code.
-	GOOS=linux $(GO) vet -mod=mod ./...
+	GOOS=$(GOOS) $(GO) vet -mod=mod ./...
```

Comment:
@iziang can you check this changes is valid?

Reply:
do get approval from @iziang

Reply:
now run 'make vet' is ok in mac arm.